app/assets/javascripts/tylium/modules/issues.js.coffee

```diff
@@ -1,5 +1,5 @@
-//= require tylium/modules/issues/evidence
-//= require tylium/modules/issues/importer
-//= require tylium/modules/issues/merge
-//= require tylium/modules/issues/table
-//= require tylium/modules/issues/tag-input
+#= require tylium/modules/issues/evidence
+#= require tylium/modules/issues/importer
+#= require tylium/modules/issues/merge
+#= require tylium/modules/issues/table
+#= require tylium/modules/issues/tag-input
```

Comment:
Just wondering, isn't the syntax `//=`?

Reply:
The asset pipeline syntax is the same, but the coffeescript syntax changed in coffeescript 4. So any coffeescript files with requires need to use this.

Reply:
Ahh okay, got it.